docs/examples/FastEmbed_Multi_GPU.ipynb

```diff
@@ -0,0 +1,126 @@
+{
```

Comment:
<p>I think this is redundant, one example (TextEmbedding) should be enough</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qdrant/fastembed/pull/422/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>